### ISSUE TITLE
Display content in horizontal slides

### ISSUE THREAD

**Issue by kezhan** [rstudio/revealjs#55]
When slide_level is 2 (by default), it seems that all level 1 will be blank pages, only with the title.  Iwould like if it is possible to display texts that are in level one. For example: 

```

---
title: "Habits"
author: John Doe
date: March 22, 2005
output: revealjs::revealjs_presentation
---

# In the morning

If I write something here, I would like to show it. Thank you

## Getting up

- Turn off alarm
- Get out of bed

```

**Comment by ZJUguquan** [comment]
the same problem...

**Comment by kazutan** [comment]
It concerns the specification of pandoc.
See https://github.com/rstudio/revealjs/issues/52.

**Comment by stevepowell99** [comment]
+1

**Comment by jthurner** [comment]
Fixed in [pandoc 2.7](https://pandoc.org/releases.html#pandoc-2.7-3-march-2019).

**Comment by apreshill** [comment]
This does appear to be fixed, thanks for the report!
